spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -90,7 +96,7 @@ def get_parser():
         '-z',
         metavar=Metavar.str,
         type=str,
-        help="Slice range to compute the metrics across (requires '-p csa'). Example: 5:23"
+        help="Slice range to compute the metrics across (requires '-p csa'). Example: 5:23"  # TODO: -p csa dose not exists anymore
```

Comment:
Note to address this TODO.

Reply:
done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3478/commits/32e3140cafffec2d72127c074c8986b6e180cf8d